cmake/OpenVDBHoudiniSetup.cmake

```diff
@@ -335,12 +335,31 @@ if(Houdini_VERSION_MAJOR_MINOR VERSION_EQUAL 18.0)
   set(OPENVDB_HOUDINI_ABI 6)
 elseif(Houdini_VERSION_MAJOR_MINOR VERSION_EQUAL 18.5)
   set(OPENVDB_HOUDINI_ABI 7)
-elseif(Houdini_VERSION_MAJOR_MINOR VERSION_EQUAL 19.0)
-  set(OPENVDB_HOUDINI_ABI 8)
 else()
-  message(WARNING "Unknown version of Houdini, assuming OpenVDB ABI=8, "
-    "but if this not correct, the CMake flag -DOPENVDB_HOUDINI_ABI=<N> can override this value.")
-  set(OPENVDB_HOUDINI_ABI 8)
+  find_file(_houdini_openvdb_version_file "openvdb/version.h"
+    PATHS ${HOUDINI_INCLUDE_DIR}
+    NO_DEFAULT_PATH
+    NO_PACKAGE_ROOT_PATH
+    NO_CMAKE_PATH
+    NO_CMAKE_ENVIRONMENT_PATH
+    NO_SYSTEM_ENVIRONMENT_PATH
+    NO_CMAKE_SYSTEM_PATH)
+  if(NOT ${_houdini_openvdb_version_file} EQUAL "_houdini_openvdb_version_file-NOTFOUND")
```

Comment:
Think this can just be `if(NOT _houdini_openvdb_version_file)`

Reply:
That's correct

> or ends in the suffix -NOTFOUND

---https://cmake.org/cmake/help/latest/command/if.html

I'll fix this as well.

Reply:
Fixed in 81a9434. Although I still needed to dereference the variable. Testing this with `cmake-3.18`.

Reply:
oops, my apologies, i think this should be `if(_houdini_openvdb_version_file)` without the `NOT` (mixed up the original statement)

Reply:
Right, that makes perfect sense. I'll fix that right away.